Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/storage/adlsgen2/SharedKeyCredential.java

```diff
@@ -48,7 +49,7 @@ public SharedKeyCredential(String accountName, String accountKey) {
         this.accountKey = Base64.getDecoder().decode(accountKey);
     }
 
-    private String buildStringToSign(HttpRequestBase request, HeaderGroup httpHeaders, List<NameValuePair> pairs) {
+    private String buildStringToSign(HttpRequestBase request, HeaderGroup httpHeaders, @Nullable List<NameValuePair> pairs) {
```

Comment:
The best practice is using `Collections.emptyList()` rather than `null` for empty list.